include/verilated_types.h

```diff
@@ -439,6 +439,20 @@ class VlQueue final {
         }
         return out;
     }
+    template <typename Func>
+    VlQueue unique(Func with_func) const {
+        VlQueue out;
+        std::unordered_set<T_Value> saw;
```

Comment:
```suggestion
        std::set<T_Value> saw;
```

Unordered set will be slower on typical small sets.